src/draw.typ

```diff
@@ -306,6 +306,30 @@
   ),)
 }
 
+/// Draw body on z-index layer
+///
+/// This can be used to draw elements behind already drawn elements by
+/// using a lower layer value (i.e -1).
+/// 
+/// - layer (integer, float): Z-Index to draw the children at
+/// - body (draw, function): Child elements or function (`ctx => draw-cmds`)
+/// - force (bool): Foce child elements on a different layer to this layer
```

Comment:
I do not understand what this means

Reply:
I removed that option. I was about moving from a layer to a different one, as the `layer` function does not overwrite an elements layer to support nesting.